python/triton/testing.py

```diff
@@ -89,14 +89,21 @@ def assert_allclose(x, y, tol=1e-2):
     assert allclose(x, y, tol)
 
 
-def random(shape, dtype, device):
-    torch.manual_seed(0)
+def random(shape, dtype, device, seed=0):
+    """
+    Override the seed in tests if you're calling this function twice and don't
+    want the same result for both calls.
+    """
+    torch.manual_seed(seed)
```

Comment:
it's indeed nicer to pass a seed as an argument. I also wonder if we should remember the seed with `torch.seed()` and revert it at the end of this function?